### DISCUSSION TITLE
How to customize e.g., bg_visual color in Lazy

### DISCUSSION THREAD

**Discussion by sdorr0** [AlexvZyl/nordic.nvim#93] [Q&A] [answered]
Thanks for this palette.  Love it!

Long time vim user, just now getting into nvim, themes, colors, etc.  New to that world.  Using lazy.  I'm not sure how to make that override (bg_visual ... the background color of text you've selected in visual mode) in my local setup (seems like you kinda have to know the right path/names/place to make that adjustment).  Any assistance would be appreciated.

Keep up the awesome work :)

**Comment by AlexvZyl** [comment, marked as answer]
Glad you like it!  Sorry for only replying now.

There is an example in the readme:

```lua
local palette = require 'nordic.colors'
require 'nordic' .setup {
    override = {
        TelescopePromptTitle = {
            fg = palette.red.bright,
            bg = palette.green.base,
            italic = true,
            underline = true,
            sp = palette.yellow.dim,
            undercurl = false
        }
    }
}
```
where `TelescopePromptTitle` is a highlight group.

You can find all the highlight groups used in [this directory's files](https://github.com/AlexvZyl/nordic.nvim/tree/main/lua/nordic/groups).

Does this help?
